extras/jackson-support/src/main/java/com/palantir/remoting3/ext/jackson/ObjectMappers.java

```diff
@@ -108,7 +108,6 @@ public static ObjectMapper newCborServerObjectMapper() {
     public static ObjectMapper withDefaultModules(ObjectMapper mapper) {
         return mapper
                 .registerModule(new GuavaModule())
-                .registerModule(new ShimJdk7Module())
```

Comment:
If this is the only piece of non-test code that was changed then in theory if people hit sad times with the new jackson, they could conceivably force back to a lower version (and re-add this) right?

Reply:
So for this code, there are two arguments:

1. This code makes remoting Jackson version independent. It's why Multipass e.g. runs 2.9 internally, so does rescue etc. So leaving it in is nice because it gives you version independence.
2. OTOH almost everything uses Jackson 2.7+ now, and we might want to use remoting to leverage that.